hexa/pipelines/management/commands/pipelines_runner.py

```diff
@@ -178,6 +190,9 @@ def run_pipeline_kube(run: PipelineRun, env_var: dict):
     except Exception:
         logger.exception("get logs")
         stdout = ""
+    # check termination reason
+    if remote_pod.status.reason == PodTerminationReason.DeadlineExceeded.value:
+        stdout = f"Timeout killed run {run.pipeline.name} #{run.id}"
```

Comment:
I'm not sure we should discard the previous value of stdout.

Also, in that case, wouldn't stdout already contain something about the timeout?

Reply:
If I remember correctly the deadlineExceeded will not be on the pod logs, but I'll double check to be more sure about it.

Reply:
Otherwise, could we try to append to `stdout` ?

Reply:
Yes, it's safer.

Reply:
@pvanliefland should be good now